babel.config.js

```diff
@@ -1,13 +1,14 @@
+// this is used by jest only
 module.exports = {
   presets: [
     [
-      '@babel/preset-env',
+      "@babel/preset-env",
       {
         targets: {
-          node: 'current'
+          node: "current"
         }
       }
     ],
-    '@babel/preset-react'
+    "@babel/preset-react"
```

Comment:
Necessary quote changes?

Reply:
Good spot, I think my editor must have done it before we move the `.prettierrc` to the root. Will fix.